net/proftpd/Portfile

```diff
@@ -32,17 +35,20 @@ master_sites        ftp://ftp.proftpd.org/distrib/source/ \
                     ftp://ftp.solnet.ch/mirror/proftpd.org/distrib/source/ \
                     gentoo
 
-depends_build       port:libtool
+# remove autoconf when updating to 1.3.10
+depends_build       port:libtool \
+                    port:autoconf
 
 depends_lib         port:libidn \
                     port:libsodium \
                     port:ncurses \
                     path:lib/libssl.dylib:openssl
 
-patchfiles          dynamic_lookup-11.patch
-
-configure.args      --sysconfdir=${prefix}/etc \
-                    --localstatedir=${prefix}/var/run \
+# remove the following two lines when updating to 1.3.10
+use_autoconf        yes
+use_configure       yes
```

Comment:
I have never seen these two used at he same time. Also `use_configure` is the default anyway. It probably should be removed.

Reply:
tested without `use_configure       yes` - removing it
🤔 in my defense, I was "using both suspenders and belt"